app/config.js

```diff
@@ -7,6 +7,7 @@ const {dialog} = require('electron');
 const isDev = require('electron-is-dev');
 const gaze = require('gaze');
 const Config = require('electron-config');
+const deepAssign = require('deep-assign'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Sorry about that, I didn't understand why I had this error occurring. The fact that is was in the wrong `package.json` explains it... :sweat_smile: